lib/matplotlib/tests/test_text.py

```diff
@@ -421,3 +421,11 @@ def test_font_scaling():
 
     for i, fs in enumerate(range(4, 43, 2)):
         ax.text(0.1, i*30, "{fs} pt font size".format(fs=fs), fontsize=fs)
+
+
+@pytest.mark.parametrize('spacing1, spacing2', [(2, 0.4), (0.4, 2)])
```

Comment:
I am very confused by what this is doing and how this relates to the added image.